test/packages/EABase/include/Common/EABase/eabase.h

```diff
@@ -114,8 +114,8 @@
 	   #define __STDC_FORMAT_MACROS
 	#endif
 	// The GCC PSP compiler defines standard int types (e.g. uint32_t) but not PRId8, etc.
-	// MSVC doesn't include an inttypes.h header.
-	#if !defined(EA_COMPILER_MSVC)
+	// MSVC added support for inttypes.h header in VS2013.
+	#if !defined(EA_COMPILER_MSVC) || EA_COMPILER_VERSION >= 1800
```

Comment:
Because 'EA_COMPILER_VERSION' is generic macro that is populated with the compilers version we need to restrict the check to MSVC.

#if !defined(EA_COMPILER_MSVC) || (defined(EA_COMPILER_MSVC) && EA_COMPILER_VERSION >= 1800)  // all other compilers and VS2013+